src/components/Topology/Typgoraphy.styles.ts

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+import fonts from '../../styles/fonts';
+import { ITypographyFontStyle, ITypographyFontType } from './types';
+
+interface ITopologyStyledProps {
+    color: string;
+    fontStyle: ITypographyFontStyle;
+    fontType: ITypographyFontType;
+}
+
+export const StyledBlock = styled.div<ITopologyStyledProps>`
```

Comment:
Minor, but shouldn't be `StyleDiv` instead due to [naming rules](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md#css-naming-an-element)? Thx